### PR TITLE
👷 Add codacy integration

### PR DESCRIPTION
**Pull request by dankolbman**
Adds codacy integration for static analysis and code coverage. See the dashboard [here](https://app.codacy.com/app/dankolbman/kf-api-dataservice/dashboard)

### CODE REVIEW

**Review by parimalak — COMMENTED** [review]
codacy is so cool 👍 . Does this fail the Circleci until we fix suggested changes or just show us the coding issues?

File: README.md [hunk 1]
```diff
@@ -3,8 +3,9 @@ Kids First Data Service
 ![Data Service](docs/dataservice.png)
 
 [![license](https://img.shields.io/github/license/kids-first/kf-api-dataservice.svg?style=for-the-badge)](https://github.com/kids-first/kf-api-dataservice/blob/master/LICENSE)
-[![CircleCI](https://img.shields.io/circleci/project/kids-first/kf-api-dataservice.svg?style=for-the-badge)](https://circleci.com/gh/kids-first/kf-api-dataservice/13?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)
 [![Docs](https://img.shields.io/readthedocs/pip.svg?style=for-the-badge)](http://kf-api-dataservice-qa.kids-first.io/)
+[![CircleCI](https://img.shields.io/circleci/project/kids-first/kf-api-dataservice.svg?style=for-the-badge)](https://circleci.com/gh/kids-first/kf-api-dataservice/13?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)
+[![Codacy grade](https://img.shields.io/codacy/grade/534528baa6d544ca9c2e2fbaad8d3a29.svg?style=for-the-badge)](https://app.codacy.com/app/dankolbman/kf-api-dataservice/dashboard)
```
Comment (allisonheath) [inline README.md]: Haven't really used codacy too much before - is there a way we can put it under the general kids-first org, or does it have to be tied to a specific github account?

**Review by allisonheath — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dankolbman) [reply README.md]: I'm not sure. It didn't seem like it from the way the setup process was laid out.

Comment (dankolbman) [reply README.md]: had to create an org under codacy

**Comment by dankolbman** [comment]
I think it will block a review if you introduce any new errors into the code. The errors can be ignored though, if they aren't real issues.

**Review by allisonheath — APPROVED** [review]

**Review by znatty22 — APPROVED** [review]
Awesome! 👍

**Pull request merged**